plugin/path_checkout_sets.go

```diff
@@ -239,9 +242,15 @@ func (b *backend) operationReserveUpdate(ctx context.Context, req *logical.Reque
 	if enforcementSent {
 		set.DisableCheckInEnforcement = disableCheckInEnforcement
 	}
+	if err := set.Validate(); err != nil {
```

Comment:
Same as previous comment re: this check coming earlier in the flow.

Reply:
I think these will be resolved by not allowing partial updates, so yay!